modules/metering-store-data-in-shared-volumes.adoc

```diff
@@ -28,15 +32,112 @@ spec:
       type: "sharedPVC"
       sharedPVC:
         claimName: "metering-nfs" <1>
-        # uncomment the lines below to provision a new PVC using the specified <2>
-        # storageClass.
+        # Uncomment the lines below to provision a new PVC using the specified storageClass. <2>
         # createPVC: true
         # storageClass: "my-nfs-storage-class"
         # size: 5Gi
 ----
 
 Select one of the configuration options below:
 
-<1> Set `storage.hive.sharedPVC.claimName` to the name of an existing ReadWriteMany PersistentVolumeClaim (PVC). This is necessary if you do not have dynamic volume provisioning or want to have more control over how the PersistentVolume is created.
+<1> Set `storage.hive.sharedPVC.claimName` to the name of an existing ReadWriteMany persistent volume claim (PVC). This is necessary if you do not have dynamic volume provisioning or want to have more control over how the persistent volume is created.
+
+<2> Set `storage.hive.sharedPVC.createPVC` to `true` and set the `storage.hive.sharedPVC.storageClass` to the name of a storage class with ReadWriteMany access mode. This will use dynamic volume provisioning to have a volume created automatically.
+--
+
+.  Create the following resource objects that are required to deploy an NFS server for metering. Use the `oc create -f <file-name>.yaml` command to create the object YAML files.
+..  Configure a `PersistentVolume` resource object. For example, `nfs_persistentvolume.yaml`:
```

Comment:
Could move the `nfs_persistentvolume.yaml` part to be a `.` title for the code block below it, if you wanted. Something like:

```
[...] resource object. For example:
+
.`nfs_persistentvolume.yaml` file
[source,yaml]
```

or

```
[...] resource object:
+
.Example `nfs_persistentvolume.yaml` file
[source,yaml]
```

(And if so, match the remaining code blocks in this procedure with this style.)

Reply:
Great suggestion. Looks cleaner with the file names as the code block example titles.